lib/Interpreter/IncrementalJIT.cpp

```diff
@@ -221,6 +222,11 @@ IncrementalJIT::getSymbolAddressWithoutMangling(llvm::StringRef Name,
     if (RuntimeDyld::SymbolInfo SymInfo = m_ExeMM->findSymbol(Name))
       return llvm::orc::JITSymbol(SymInfo.getAddress(),
                                   llvm::JITSymbolFlags::Exported);
+#ifdef LLVM_ON_WIN32
+    if (const void* Sym = platform::DLSym(Name))
```

Comment:
Why doesn't the process symbol resolution (in `RTDyldMemoryManager::getSymbolAddressInProcess()`) work, why do we need a special Windows code path here?